### PR TITLE
Explore comprehensive test

### PR DESCRIPTION
**Pull request by spathare21**
JIRA issue URL: *paste URL here*

*[Add a short description of what is changed here.]*

## Reviewer tasks

The following is based on the
[Zanata Development Guidelines](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines).
As a reviewer, you should check the guidelines regularly for updates, and just
to refresh your memory.

The reviewer can use this list for reference to make sure
they have considered all the important points.

Use the checkboxes or not, whatever works best for you.

 - Code quality
   - [ ] Code passes style check (checkstyle/eslint)
   - [ ] Code is clear and concise
   - [ ] UI code is internationalised
   - [ ] Code has adequate comments where appropriate
   - Code is in an appropriate place
     - [ ] Classes are in appropriate packages
     - [ ] Code fits with class's responsibilities (SRP)
   - [ ] Configuration files are documented enough
   - If code is removed
     - [ ] No remaining code references the removed code
       - [ ] No orphaned rewrite rules
       - [ ] No orphaned config settings
     - [ ] No remaining comments refer to the removed code
     - [ ] Unused imports and libraries are removed
 - Testing
   - [ ] New code has tests
   - [ ] Changed code has tests
   - [ ] All tests pass
   - [ ] Test coverage stable or increased
 - Documentation
   - [ ] New features are documented
   - [ ] Docs removed for deleted features
   - [ ] Docs updated for all changed features
   - [ ] Developer/sysadmin docs updated if appropriate
 - If there are new dependencies
   - [ ] Does each new dependency pass all the
         [Considerations for new dependencies](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines#new-technologies-and-dependencies)?

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-platform/394)

### CODE REVIEW

File: server/functional-test/src/test/java/org/zanata/feature/search/comp/GroupSearchCTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                .as("Normal user can see the group listed");
+
+        VersionGroupPage versionGroupPage = explorePage.clickGroupSearchEntry(groupName);
+
+        assertThat(versionGroupPage.getGroupName().trim())
+                .isEqualTo(groupName);
+    }
+
+    @Test(timeout = ZanataTestCase.MAX_SHORT_TEST_DURATION)
+    public void unsuccessfullGroupSearch() throws Exception {
```
Comment (djansen-redhat) [inline server/functional-test/src/test/java/org/zanata/feature/search/comp/GroupSearchCTest.java]: Only one L in unsuccessful.

File: server/functional-test/src/test/java/org/zanata/feature/search/comp/LanguageSearchCTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015, Red Hat, Inc. and individual contributors as indicated by the
```
Comment (djansen-redhat) [inline server/functional-test/src/test/java/org/zanata/feature/search/comp/LanguageSearchCTest.java]: 2017

File: server/functional-test/src/test/java/org/zanata/feature/search/comp/LanguageSearchCTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author Sachin Pathare <a
+ *         href="mailto:spathare@redhat.com">spathare@redhat.com</a>
+ */
+@Category(TestPlan.ComprehensiveTest.class)
+public class LanguageSearchCTest extends ZanataTestCase {
+
+    @Test(timeout = ZanataTestCase.MAX_SHORT_TEST_DURATION)
```
Comment (djansen-redhat) [inline server/functional-test/src/test/java/org/zanata/feature/search/comp/LanguageSearchCTest.java]: "ZanataTestCase." should not be necessary

File: server/functional-test/src/test/java/org/zanata/feature/search/comp/LanguageSearchCTest.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    public void successfullLangSearch() throws Exception {
```
Comment (djansen-redhat) [inline server/functional-test/src/test/java/org/zanata/feature/search/comp/LanguageSearchCTest.java]: Only one L in successful.

File: server/functional-test/src/test/java/org/zanata/feature/search/comp/LanguageSearchCTest.java [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+                .gotoExplore()
+                .enterSearch("eng")
+                .expectLanguageTeamListContains("English (United States)");
+
+        assertThat(explorePage.getLanguageSearchResults())
+                .contains("English (United States)")
+                .as("Normal user can see the languages listed");
+    }
+
+    @Test(timeout = ZanataTestCase.MAX_SHORT_TEST_DURATION)
```
Comment (djansen-redhat) [inline server/functional-test/src/test/java/org/zanata/feature/search/comp/LanguageSearchCTest.java]: "ZanataTestCase." should not be necessary

File: server/functional-test/src/test/java/org/zanata/feature/search/comp/LanguageSearchCTest.java [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    public void unsuccessfullLangSearch() throws Exception {
```
Comment (djansen-redhat) [inline server/functional-test/src/test/java/org/zanata/feature/search/comp/LanguageSearchCTest.java]: Only one L in unsuccessful.

**Comment by spathare21** [comment]
@zanata-jenkins retest this please

**Pull request merged**